db/migrate/20180219192755_create_trips.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTrips < ActiveRecord::Migration[5.1]
+  def change
+    create_table :trips do |t|
+      t.integer :duration
+      t.datetime :start_date
+      t.datetime :end_date
+      t.integer :bike_id
+      t.string :subscription_type
+      t.integer :zip_code
```

Comment:
We will need some way to parse the zip code to five digits (as well as missing values) when we seed